### DISCUSSION TITLE
How to mount and boot kali on samsung

### DISCUSSION THREAD

**Discussion by daniela2000111** [community/community#142212] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

1:usb
2.rufus

**Comment by Chanduporalla** [comment]
To mount and boot Kali Linux on a Samsung device, you'll need to follow a process that involves installing Kali Linux on either a smartphone or tablet using various methods such as chroot, termux, or full boot via custom recovery (for rooted devices). Below is a general guide for two common approaches: using Termux (without root) and using custom recovery (with root access).
Method 1: Install Kali Linux Using Termux (No Root Required)

    Install Termux and Nethunter Kex:
        Download and install Termux from the Google Play Store or F-Droid.
        Open Termux and run the following commands to update packages and install dependencies:

        bash

    pkg update && pkg upgrade
    pkg install wget curl proot tar -y

Install Kali Linux ARM:

    Download the Kali Linux installer script for Termux:

    bash

wget -O install-nethunter-termux https://offs.ec/2MceZWr

Change the script's permissions to make it executable:

bash

chmod +x install-nethunter-termux

Run the installer:

bash

    ./install-nethunter-termux

This will download and install the Kali Linux base system inside Termux.

Start the Kali Environment:

    After installation, start Kali Linux using:

    bash

    nh

This will boot you into a Kali Linux environment within Termux.

Install NetHunter Kex (Graphical Interface):

    To enable a graphical interface, install NetHunter Kex:

    bash

apt update && apt install -y kali-win-kex

After installation, start the VNC server to access a full graphical desktop:

bash

        kex

        Download a VNC viewer (like RealVNC) on your Samsung device, and connect using the IP address 127.0.0.1 with the default password (kali).

You now have a fully functional Kali Linux running inside Termux on your Samsung device without root.
Method 2: Install Kali Linux via Custom Recovery (Root Required)

If you want a full bootable Kali Linux on your Samsung device, you’ll need root access and a custom recovery like TWRP.

    Root Your Samsung Device:
        Unlock the bootloader of your Samsung device (this process varies depending on the model).
        Flash a custom recovery such as TWRP (TeamWin Recovery Project) on your device. You can find a TWRP guide specific to your device model.

    Download the Kali Linux ARM Image:
        Download the Kali Linux ARM image for your architecture from the official site: Kali ARM Images.
        You can either download it directly on your device or on a computer and transfer it to your Samsung device.

    Flash the Kali Linux Image:
        Boot into TWRP recovery mode.
        In TWRP, navigate to Install and locate the Kali Linux image that you downloaded.
        Flash the image and reboot your device.

    Boot into Kali Linux:
        After flashing, boot into Kali Linux by selecting it from the bootloader (if dual boot is set up) or boot directly if Kali is your primary OS.
but Using Termux is a great option

**Comment by LiteBrite82** [comment]
Hi @daniela2000111 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by hh9733190-commits** [comment]
ok but when I install nethunter kex it send to me massage "this app for older android version"
